lib/loader.js

```diff
@@ -30,8 +39,8 @@ module.exports = function(content, map) {
 
 	processCss(content, map, {
 		mode: moduleMode ? "local" : "global",
-		from: loaderUtils.getRemainingRequest(this),
-		to: loaderUtils.getCurrentRequest(this),
+		from: loaderUtils.getRemainingRequest(this).split("!").pop(),
```

Comment:
Look into `resourcePath`, yup. I can't remember the exact API, though. Maybe this needs some guidelines.